crates/burn-tensor/src/tests/quantization/ops/ceil.rs

```diff
@@ -0,0 +1,24 @@
+#[burn_tensor_testgen::testgen(q_ceil)]
+mod tests {
+    use super::*;
+    use burn_tensor::quantization::{AffineQuantization, QuantizationStrategy};
+    use burn_tensor::{Tensor, TensorData};
+
+    #[test]
+    fn should_support_ceil_ops() {
+        let data = TensorData::quantized(
```

Comment:
Could you add the original float values represented by the quantized tensor data in a comment? It's useful to get a quick glance at the test values.